pkg/liveshare/session.go

```diff
@@ -123,6 +123,20 @@ func (s *Session) StartJupyterServer(ctx context.Context) (int, string, error) {
 	return port, response.ServerUrl, nil
 }
 
+func (s *Session) Rebuild(ctx context.Context) error {
+	var rebuildSuccess bool
+	err := s.rpc.do(ctx, "IEnvironmentConfigurationService.rebuildContainer", []string{}, &rebuildSuccess)
```

Comment:
Nit: nil is an empty array so you can pass that for args instead

Reply:
Replace `[]string{}` with `nil`?